transaction.go

```diff
@@ -162,7 +162,7 @@ func (t *Transaction) Argument(i int) (cadence.Value, error) {
 
 	encodedArg := t.Arguments[i]
 
-	arg, err := jsoncdc.Decode(encodedArg)
+	arg, err := jsoncdc.Decode(nil, encodedArg)
```

Comment:
The CI error is weird: The function is called with 2 arguments, but it's (incorrectly?) reporting it's too many. However, in the test above, the function accepts 2 arguments. 🤔

Reply:
This is because these changes require the secure cadence version, and CI is running on the latest actual release. Once we have an m4 release of Cadence we can update the go.mod here and CI should work.

Reply:
Ah, I see 👍